### DISCUSSION TITLE
SOLVED MOSTLY: Accuracy depending <> print resolution <> layer height

### DISCUSSION THREAD

**Discussion by eefweenink** [sn4k3/UVtools#641] [Q&A] [answered]
Searched around on the forum, and maybe it was allready explained and I missed it. But anyhow this is my question. 
I bought a Creality CL60, to be delivered end of the month. 
It has a LCD with 2560 x 1620 pixels resolution, this gives a print granularity of ~50 microns,
Layer height can be set lower then that. ( If I read well minimum is about 10um.) 

1. So now I try to understand how to work with the maximum detail possible. If I set layer heighth lower then 50um, can that be usefull or just lost of time? 
2. What is the most accurate? Is there difference because of these settings? . One of the first things to be done is to run some test, and get the micrometer out and check what results we have. 

If this is allready discussed elsewhere, please give me a link to that discussion. 
Thank you very much, 
Regards, Eef

**Comment by eefweenink** [comment, marked as answer]
Hitting the HELP in UVTools I found this one: https://ameralabs.com/blog/the-complete-resin-3d-printing-settings-guide-for-beginners. With most answers I could hope for. Sorry to have disturbed you. ;- Eef

**Reply by sn4k3** [reply]
As for minimun layer height I don't recommend lower than 0.3mm
